graylog2-web-interface/src/pages/ConfigurationsPage.jsx

```diff
@@ -29,12 +29,14 @@ const EVENTS_CONFIG = 'org.graylog.events.configuration.EventsConfiguration';
 const URL_WHITELIST_CONFIG = 'org.graylog2.system.urlwhitelist.UrlWhitelist';
 
 class ConfigurationsPage extends React.Component {
-  state = {
-    loaded: false,
-  }
-
   checkLoadedTimer = undefined
 
+  constructor() {
+    super();
```

Comment:
Don't we need to pass `props` through?

```suggestion
  constructor(props) {
    super(props);
```

Reply:
Right! I always forget that. Been aged since I used component classes 😅